### ISSUE TITLE
Generated template extensions class should not be in default package

### ISSUE THREAD

**Issue by datho7561** [redhat-developer/quarkus-ls#698]
As of now, when you generate a template extension in a new template extension class for a missing member in a qute template, the generated class is always put in the default package. It would be nice to put it somewhere other than the default package.

**Comment by datho7561** [comment]
Fred mentioned that a wizard would be a good idea in order to indicate where to put the new class

**Comment by angelozerr** [comment]
> Fred mentioned that a wizard would be a good idea in order to indicate where to put the new class

Indeed I think it is a better idea. I think we could start that in vscode-quarkus but perhaps vscode-java should provide this wizard (@rgrunber @fbricon  what do you think about that?) that vscode-quarkus could use it by pre-filling package and TemplateClass name.

**Comment by fbricon** [comment]
vscode-java doesn't provide such wizard

**Comment by angelozerr** [comment]
Yes I know but my idea is to implement this wizard on vscode quzrkus side and move it to vscode java.
